docs/10_using_dandi.md

```diff
@@ -113,10 +113,24 @@ two different servers differ slightly.
       After you provide name and description, the dataset identifer will be created, 
       we will call this `<dataset_id>`.
     1. NWB format:
-        1. Convert your data to NWB 2.1+ in a local folder. Let's call this `<source_folder>`
+        1. Convert your data to NWB 2.1+ in a local folder. Let's call this `<source_folder>`.
+        We suggest beginning the conversion process using only a small amount of data so that common issues may be spotted earlier in the process.
         This step can be complex depending on your data. Feel free to [reach out to
         us for help](https://github.com/dandi/helpdesk/discussions).
-        1. Validate the NWB files by running: `dandi validate <source_folder>`. 
+        1. Check your files for [NWB Best Practices](https://nwbinspector.readthedocs.io/en/dev/best_practices/best_practices_index.html) by installing
+        the [NWBInspector](https://nwbinspector.readthedocs.io/en/dev/user_guide/user_guide.html) (`pip install -U nwbinspector`) and running
+
+                nwbinspector <source_folder> --config dandi
+                
+        1. Thoroughly read through the NWBInspector report and try to address as many issues as possible. **DANDI will prevent validation and upload of any issues
+        labeled as level 'CRITICAL' or above when using the `--config dandi` option.**
```

Comment:
This part, of course, depends on whatever we decide in [PR #941](https://github.com/dandi/dandi-cli/pull/941)

Reply:
And was indeed the accepted version.